studio-docs/source/github-integration.md

```diff
@@ -53,7 +54,29 @@ jobs:
             apollo service:push
           fi
 ```
+### Github Actions
+```
+name: Check
+
+# Controls when the action will run. Triggers schema validation on every pull request against master
+on:
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  schema-check:
+    # The type of runner that the job will run on any runner with node will work
+    runs-on: ubuntu-latest
 
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run schema check
+      env: # Apollo Key is supplied as an env var from github secrets
+        APOLLO_KEY: ${{ secrets.APOLLO_KEY }}
+      run: npx apollo service:check --branch=${GITHUB_REF#refs/heads/} --author=$GITHUB_ACTOR
```

Comment:
Should we include `--localSchemaFile`?

Reply:
We don't in the above, so I'm leaning towards leaving it out. We include a section below about how to get the graphql document